app/modules/intelligence/agents_copy/chat_agents/adaptive_agent.py

```
@@ -0,0 +1,114 @@
+        self.prompt_provider = prompt_provider
+        self.agent_type = agent_type
+
+        self.rag_agent = rag_agent
+
+    async def _create_llm(self) -> ChatAgent:
+        llm_prompts = await self.prompt_provider.get_prompts_by_agent_id_and_types(
+            str(self.agent_type), [PromptType.SYSTEM, PromptType.HUMAN]
+        )
+        prompts = {prompt.type: prompt for prompt in llm_prompts}
+        system_prompt = prompts.get(PromptType.SYSTEM)
+        if system_prompt == None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)